Remove two stderr messages that Scintilla has in it's mac code base.

Index: /home/toddw/as/komodo-devel/contrib/scintilla/macosx/ScintillaMacOSX.cxx
===================================================================
--- macosx/ScintillaMacOSX.cxx	(revision 58962)
+++ macosx/ScintillaMacOSX.cxx	(working copy)
@@ -1485,17 +1485,16 @@
 void ScintillaMacOSX::Copy()
 {
     if (!sel.Empty()) {
 #ifdef EXT_INPUT
         ExtInput::stop (GetViewRef());
 #endif
         SelectionText selectedText;
         CopySelectionRange(&selectedText);
-        fprintf(stderr, "copied text is rectangular? %d\n", selectedText.rectangular);
         CopyToClipboard(selectedText);
     }
 }
 
 bool ScintillaMacOSX::CanPaste()
 {
     if (!Editor::CanPaste())
         return false;
@@ -1521,17 +1520,16 @@
 {
     PasteboardRef theClipboard;
     SelectionText selectedText;
     selectedText.rectangular = forceRectangular;
     bool isFileURL = false;
     PasteboardCreate( kPasteboardClipboard, &theClipboard );
     bool ok = GetPasteboardData(theClipboard, &selectedText, &isFileURL);
     CFRelease( theClipboard );
-    fprintf(stderr, "paste is rectangular? %d\n", selectedText.rectangular);
     if (!ok || !selectedText.s)
         // no data or no flavor we support
         return;
 
     pdoc->BeginUndoAction();
     ClearSelection();
     if (selectedText.rectangular) {
         SelectionPosition selStart = sel.RangeMain().Start();
